exporter/prometheusremotewriteexporter/metadata.yaml

```diff
@@ -0,0 +1,7 @@
+type: prometheusremotewrite
+
+status:
+  class: exporter
+  stability:
+    beta: [metrics]
+  distributions: [core, contrib, aws]
```

Comment:
what does core mean? Is that correct?

Reply:
I supposed core is this distribution? https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/manifest.yaml#LL27C24-L27C91

Reply:
https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol